data/human/transport missions.txt

```diff
@@ -3582,3 +3582,297 @@ mission "Hauler VI cargo prototype 3"
+	to offer
+		has "main plot completed"
+		or
+			has "Intro [3 Transport]: done"
+			has "Intro [3 Freighter]: done"
+			has "Intro [3 Interceptor]: done"
+	on offer
+		conversation
+			`Finding yourself on <origin>, you smile at the memory of having dropped James here after he gave you some basic spaceflight instructions. Do you wish to visit him and see how he has been faring?`
+			choice
+				`	(Yes.)`
+				`	(Not right now.)`
+					defer
+			`	Unlike some other planets, where it is practically impossible to find a person without having contact information, <origin> has a planetary database. You eagerly walk over to the nearest information desk and wait for the operator to make a few calls. At last, you are directed to a small, respected establishment, the Pilot Veteran Clinic. You can only wonder what James is doing there.`
+			`	The clinic is housed in a small, modern-looking building, which looks cunningly similar to a giant Firebird. Inside, the rooms are arranged to resemble bunk rooms on a spaceship, likely so that the patients would feel right at home.`
+			`	As it is currently lunch time, there is no one in sight at the clinic. Luckily, the operator gave you the room number, which you quickly reach after a few minutes.`
+			choice
+				`	(Wait for someone to show up.)`
+					goto wait
+				`	(Enter the room.)`
+			apply
+				set "james medicine: infected"
+			`	You enter the room and find that James is inside, lightly asleep. He wakes up just as you close the door.`
+			`	"<first>! Fancy meeting you here! I heard you became a war hero of sorts?", he adds with a wink.`
+			choice
+				`	"Nothing special, I just did what is right."`
+				`	"Indeed, seems like I saved the galaxy this time."`
+			`	"Yeah, yeah... You know, I am feeling pretty weak now, and while the doctors say that I will live through this illness, there is still a chance that it would be my last."`
+			`	"So, I wanted to tell you something..", he rambles on, " just to make sure that knowledge is not lost. While I was exploring the planets and systems reachable from human space, I stumbled upon Ultima Thule, due northeast of here. At the time I piloted just a small warship, so I couldn't muster the courage to pass through it, but.. there is a wormhole there!"`
+			branch interesting
+				not "First Contact: Hai: offered"
+			choice
+				`	(Tell him about the Hai.)`
+				`	(Say nothing about it.)`
+					goto interesting
+			`	"I have actually been there," you say, "It leads to a nearby part of space inhabited by the Hai, an advanced alien civilization of squirrel-like creatures."`
+			`	"Indeed? Are they friendly?"`
+			choice
+				`	"Yes, they even offer humans permission to settle there or buy their technology."`
+					goto doctor
+				`	"Yes, though I'm worried they can conquer all of human space if they wish."`
+					goto doctor
+			label interesting
+			`	"Sounds interesting," you say, "Could it lead to some aliens, or perhaps to some worlds waiting to be settled?"`
```

Comment:
might want to use
```suggestion
			`	"Sounds interesting," you say, "Could it lead to some aliens, or perhaps to some worlds waiting to be colonized?"`
```
settle is more of an individual action.

Reply:
Seems like I couldn't think of that word in time when writing it. Thanks :)